pom.xml

```diff
@@ -222,6 +222,26 @@
             <plugin>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>${compiler-plugin.version}</version>
+                <executions>
+                    <execution>
+                        <phase>compile</phase>
+                        <goals>
+                            <goal>compile</goal>
+                        </goals>
+                    </execution>
+                    <execution>
```

Comment:
Isn't this what https://github.com/RedHatInsights/notifications-backend/blob/master/pom.xml#L262 already does?

Reply:
Ah no, that above creates the extension, so that L262+ can do its work?

Reply:
That's correct, this run before the avro schema to compile required java files.
In the future, this could be a separate maven module to share with others java projects.